src/main/resources/application.yaml

```diff
@@ -1,4 +1,6 @@
 spring:
+    codec:
+        max-in-memory-size: 10MB
```

Comment:
Hvordan har du kommet frem til 10MB?

Reply:
Det var en finger opp i lufta og en antagelse siden jeg ikke har så voldsomt lyst til å sette unlimited.
Jeg og Espen regna på det. Vi må egentlig sette den til 85 MB for å dekke alle BC i dag, så vi må tenke oss litt om vi skal øke den ytterligere